doc/networking.md

```diff
@@ -77,7 +79,19 @@ admin@example:/config/interfaces/interface/eth0.20/> set vlan lower-layer-if eth
 admin@example:/config/interfaces/interface/eth0.20/> leave
 ```
 
-> **Note:** If you name your VLAN interface `foo0.N`, where `N` is a number, Infix will set the interface type automatically for you.
+The example below assumes bridge br0 is already created.
+
+```
+admin@example:/> configure 
+admin@example:/config/> edit interfaces interface vlan10
+admin@example:/config/interfaces/interface/vlan10/> set vlan id 10
+admin@example:/config/interfaces/interface/vlan10/> set vlan lower-layer-if br0
+admin@example:/config/interfaces/interface/vlan10/> leave
+```
+
+As conventions, a VLAN interface for VID 20 on top of an Ethernet interface *eth0* is named *eth0.20*, and a VLAN interface for VID 10 on top of a bridge interface *br0* is named *vlan10*. 
+
```

Comment:
Maybe add something like: "Remember to configure the VLAN filtering of the bridge."

Reply:
New commits with reference to the VLAN Filtering Bridge section. In addition, that section has been extended.